WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -141,6 +157,22 @@ private extension ProductsViewController {
             "Products",
             comment: "Title that appears on top of the Product List screen (plural form of the word Product)."
         )
+
+        navigationItem.leftBarButtonItem = {
+            let button = UIBarButtonItem(image: .searchImage,
+                                         style: .plain,
+                                         target: self,
+                                         action: #selector(displaySearchProducts))
+            button.tintColor = .white
```

Comment:
Use `StyleManager.wooWhite` here. We'll be adding dark mode and the StyleManager will have semantic colors.